src/Symfony/Component/Mailer/Bridge/Postmark/Transport/PostmarkApiTransport.php

```diff
@@ -51,13 +54,26 @@ public function __toString(): string
 
     protected function doSendApi(SentMessage $sentMessage, Email $email, Envelope $envelope): ResponseInterface
     {
-        $response = $this->client->request('POST', 'https://'.$this->getEndpoint().'/email', [
-            'headers' => [
-                'Accept' => 'application/json',
-                'X-Postmark-Server-Token' => $this->key,
-            ],
-            'json' => $this->getPayload($email, $envelope),
-        ]);
+        try {
+            $response = $this->client->request('POST', 'https://'.$this->getEndpoint().'/email', [
+                'headers' => [
+                    'Accept' => 'application/json',
+                    'X-Postmark-Server-Token' => $this->key,
+                ],
+                'json' => $this->getPayload($email, $envelope),
+            ]);
+        } catch (HttpTransportException $e) {
+            if (
+                str_contains($e->getMessage(), 'Unable to send an email: You tried to send to recipient(s) that have been marked as inactive.')
+                && null !== $this->messageBus
+                && class_exists(ConsumeRemoteEventMessage::class)
+            ) {
+                // Postmark throws an exception when sending an email to an inactive email address
+                $this->messageBus->dispatch(new ConsumeRemoteEventMessage('default', new MailerDeliveryEvent(MailerDeliveryEvent::DROPPED, 'MessageID', [])));
```

Comment:
I don't know how to correctly instantiate this object.

When looking at `WebhookController`, the type is parsed from the config file, so probably hard to access the right one here.

I can't either get the MessageID from the response, as my IDE warns me that it is potentially undefined, and I don't have a similar payload as I would have in a webhook call.

🤔

Reply:
if postmark refuses to send a message due to marking a recipient as inactive, there is probably no message id at all (as they haven't generated a message yet)